### PR TITLE
Updated notebook setup guide for Kubeflow 1.0.0 UI

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1761
Fixes https://github.com/kubeflow/website/issues/1507

A number of options have changed in the notebook setup for Kubeflow 1.0.0. This PR updates the setup guide accordingly.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1762)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1762--competent-brattain-de2d6d.netlify.com/docs/notebooks/setup/

@kimwnasptd Please would you review this PR and LGTM if you're happy with the updates?
/cc @kimwnasptd

**Comment by sarahmaddox** [comment]
/assign @abhi-g 

@abhi-g Could you take a look at this PR or assign someone else to do so? I'm not having much luck finding a reviewer. :)

**Comment by kimwnasptd** [comment]
Sorry for the delay @sarahmaddox taking a look right now :)

File: content/docs/notebooks/setup.md [hunk 1]
```diff
@@ -86,34 +86,38 @@ central dashboard](/docs/components/central-dash/overview/).
...
+            ```
+            kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"
+            ```
           * If you have GPUs available, you can schedule your server on a GPU node 
-            in the **Extra Resources** section at the bottom of the form. For 
-            example, to reserve two GPUs, enter the following JSON code: 
-          `{"nvidia.com/gpu": 2}`
-      * Kubeflow version (for example, `v0.5.0`).
-    
+            in the **GPUs** section at the bottom of the form.
```
Comment (kimwnasptd) [inline content/docs/notebooks/setup.md]: Would it make more sense to rephrase the `you can schedule your server on a GPU node in the GPUs section at the bottom of the form` to something like:

`you can schedule your server on a GPU node by specifying a number of GPUs to be attached to the server under the **GPUs** section at the bottom`

**Comment by sarahmaddox** [comment]
/assign @kimwnasptd 
/unassign @abhi-g 

Thanks @kimwnasptd PTAL

**Comment by sarahmaddox** [comment]
Thanks @kimwnasptd !
/approve

**Pull request merged**